### DISCUSSION TITLE
Collab server: end-to-end encryption

### DISCUSSION THREAD

**Discussion by naim94a** [zed-industries/zed#16437] [Feature Requests]
First, I want to thank you for creating such an amazing code editor. I’ve been enjoying using Zed, and the collaboration features are especially helpful for team projects.

I’d like to request the implementation of end-to-end encryption (E2EE) for communications with the collab server. Ensuring that all data shared between users is encrypted and inaccessible to third parties, including the server itself, would greatly enhance the security and privacy of collaborative work using Zed.

**Comment by AdoreIt** [comment]
Hello! Voting for this as well!
